### PR TITLE
Cleanups: Reduce chain specs, docs, path filters

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is to separate out several small fixes I found while working on #870 

Part of #870 

# Discussion
- Corrected the path filters for integration tests
- Remove (now unused) method used to generate the chain spec for frequency-rococo
- Merge function with parent that just called it
- Documentation

### CODE REVIEW

File: node/service/src/chain_spec.rs [hunk 1]
```diff
@@ -13,10 +13,7 @@ pub type DummyChainSpec = sc_service::GenericChainSpec<(), Extensions>;
 #[cfg(feature = "frequency")]
 pub mod frequency;
 
-#[cfg(feature = "frequency-rococo-local")]
-pub mod frequency_local;
-
-#[cfg(feature = "frequency-rococo-testnet")]
+#[cfg(any(feature = "frequency-rococo-testnet", feature = "frequency-rococo-local"))]
 pub mod frequency_rococo;
```
Comment (wilwade) [inline node/service/src/chain_spec.rs:17]: Local is still a "rococo" so now they use the same setup

File: node/service/src/chain_spec/frequency_rococo.rs [hunk 2]
```diff
@@ -1,183 +1,111 @@
...
-	pub const FRQ_COUNCIL10: &str =
-		"0x1caccabbe8095a35292782cffc29145030264f9706753923a5c237e8f5aceb1a";
+/// Generate the session keys from individual elements.
+///
+/// The input must be a tuple of individual keys (a single arg for now since we have just one key).
+pub fn template_session_keys(keys: AuraId) -> frequency_runtime::SessionKeys {
+	frequency_runtime::SessionKeys { aura: keys }
 }
 
-pub fn frequency_rococo_testnet() -> ChainSpec {
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency_rococo.rs:60]: This is the function that was used to generate the genesis for the Frequency Rococo (XRQCY) testnet. It is now unused and running it "locally" or against a different Rococo requires using the `local_testnet_config`

File: node/service/src/chain_spec/frequency_rococo.rs [hunk 3]
```diff
@@ -1,183 +1,111 @@
...
+/// Generates the chain spec for a local testnet
+pub fn local_testnet_config() -> ChainSpec {
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency_rococo.rs:32]: This is just moved code from `node/service/src/chain_spec/frequency_local.rs`

File: node/service/src/service.rs [hunk 4]
```diff
@@ -590,14 +593,3 @@ fn frequency_dev_instant(
...
-
-/// Function to start frequency parachain with instant sealing in dev mode.
-/// This function is called when --chain dev --instant-sealing is passed.
-/// This function is called when --chain frequency_dev --instant-sealing is passed.
-#[cfg(feature = "frequency-rococo-local")]
-pub fn frequency_dev_instant_sealing(
-	config: Configuration,
-	is_instant: bool,
-) -> Result<TaskManager, sc_service::error::Error> {
-	frequency_dev_instant(config, is_instant)
```
Comment (wilwade) [inline node/service/src/service.rs:602]: Removed stack frame

**Review by saraswatpuneet — APPROVED** [review]
:broom:

File: node/cli/src/cli.rs [hunk 5]
```diff
@@ -77,12 +77,18 @@ pub struct Cli {
...
 	/// Instant block sealing
+	/// Blocks are triggered to be formed each time a transaction hits the validated transaction pool
+	/// Empty blocks can also be formed using the `engine_createBlock` RPC
 	///
 	/// Can only be used with `--dev` and turned on with frequency-rococo-local feature flag
 	#[cfg(feature = "frequency-rococo-local")]
 	#[clap(long = "instant-sealing", requires = "dev")]
 	pub instant_sealing: bool,
 
+	/// Manual block sealing
```
Comment (enddynayn) [inline node/cli/src/cli.rs:88]: 👍

File: node/service/src/chain_spec/frequency_rococo.rs [hunk 6]
```diff
@@ -1,183 +1,111 @@
...
-					// 5CXnMFFgruHxAFtvCogwm8TFygWg1MWd9KhMnfEPbRdCpf74
-					public_testnet_keys::FRQ_COUNCIL7.parse::<AccountId>().unwrap().into(),
-					// 5CWzQaAJFqYoF1bZWsvEnzMQDGokk2csTFBwfPpo1oNfBGkn
-					public_testnet_keys::FRQ_COUNCIL8.parse::<AccountId>().unwrap().into(),
-					// 5HmcreGLq25iA7fiyb6An4YVWC3k1Cq8SQgYn2Qpeepq24nV
-					public_testnet_keys::FRQ_COUNCIL9.parse::<AccountId>().unwrap().into(),
-					// 5CiJXZxsko8YZGhTZAwrhREUzjCVEGwNE7Fgrf1twbt8WsmY
-					public_testnet_keys::FRQ_COUNCIL10.parse::<AccountId>().unwrap().into(),
-					// Treasury $$
+					get_account_id_from_seed::<sr25519::Public>("Alice"),
```
Comment (enddynayn) [inline node/service/src/chain_spec/frequency_rococo.rs:60]: nice!

**Review by enddynayn — APPROVED** [review]
👍 fabulous!

**Pull request merged**